### PR TITLE
Added Buttons class

### PR DESCRIPTION
**Pull request by swensenkailen**
More compact version of Button class used in button-test, made for a circuit with four buttons and defaulted to fit our prototype model.

### CODE REVIEW

**Comment by ilikedturtles** [comment]
It looks like you're initializing the buttons at global construction time, which may occur before all prerequisites are met.  I'm not sure that it will be an issue, but I'd feel more comfortable saying it looks good if the initialization had to be manually invoked by the user in setup().

File: Buttons.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+class Buttons
+{
+  public:
+
+  // defaulted to demo circuit values
+  Buttons() : gap(500), pin1(10), pin2(11), pin3(12), pin4(13)
+  {
+    setup();
```
Comment (ilikedturtles) [inline Buttons.h]:
```suggestion

```

File: Buttons.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  void setup()
```
Comment (ilikedturtles) [inline Buttons.h]:
```suggestion
  void Init()

```

**Review by ilikedturtles — CHANGES_REQUESTED** [review]
Initialization should be manually invoked in setup() function of theremin.ino

File: Buttons.h (hunk 1 above)
Comment (swensenkailen) [reply Buttons.h]: Ah yes, that makes sense. I'll resolve with your changes. Thanks for catching that!

**Pull request merged**